### ISSUE TITLE
pagination-ellipsis.svelte: The value of 'aria-hidden' must be either 'true' or 'false'. It cannot be empty

### ISSUE THREAD

**Issue by mustafa0x** [huntabyte/shadcn-svelte#1510]
### Describe the bug

vite/svelte-warn prints this:

```
pagination-ellipsis.svelte:2:4 The value of 'aria-hidden' must be either 'true' or 'false'. It cannot be empty
```

### Reproduction

https://github.com/huntabyte/shadcn-svelte/blob/main/sites/docs/src/lib/registry/default/ui/pagination/pagination-ellipsis.svelte#L13

**Comment by ieedan** [comment]
This is fixed on `next`. You can fix it in your code by adding `=true`

```svelte
<span
	aria-hidden="true"
	class={cn("flex h-9 w-9 items-center justify-center", className)}
	{...$$restProps}
>
	<DotsHorizontal class="h-4 w-4" />
	<span class="sr-only">More pages</span>
</span>
```

**Comment by mustafa0x** [comment]
Ah, I thought I checked next, but I forgot next isn't the default branch.
